Resources/Prototypes/Entities/Objects/Fun/dice.yml

```diff
@@ -131,3 +131,81 @@
       types:
         Piercing: 5
     # I love this
+
+
+- type: entity
+  parent: PercentileDie
+  id: PercentileDieLoaded
+  suffix: loaded
+  components:
+  - type: LoadedDice
+  - type: UserInterface
+    interfaces:
+      enum.LoadedDiceUiKey.Key:
+        type: LoadedDiceBoundUserInterface
+
+- type: entity
+  parent: d20Dice
+  id: d20DiceLoaded
+  suffix: loaded
+  components:
+  - type: LoadedDice
+  - type: UserInterface
+    interfaces:
+      enum.LoadedDiceUiKey.Key:
+        type: LoadedDiceBoundUserInterface
+
```

Comment:
you can do separate parent with both components and suffix and then write `parent: [d20Dice, BaseDiceLoaded]`
it will reduce yml lines